database/migrations/2023_07_29_154546_remove_prototype_elephpants_from_collections.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class RemovePrototypeElephpantsFromCollections extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::statement('DELETE FROM elephpant_user WHERE elephpant_id IN (52,53,55,62,77)');
```

Comment:
```suggestion
        DB::statement("
            DELETE 
            FROM elephpant_user eu
            LEFT JOIN users u ON eu.user_id = u.id
            WHERE elephpant_id IN (52,53,55,62,77)
            AND u.twitter != 'OGProgrammer';
        ");
```
@OGProgrammer is active on the site and owns the prototype for 62.  His herd doesn't list any other prototypes so I've kept the query simple: delete prototypes from all collections except his